ui/src/modules/app_builder/Canvas/index.tsx

```diff
@@ -41,6 +41,7 @@ export interface BuilderCanvasProps {
   onClick?: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void
   interaction?: InteractionInfo
   errorInteraction?: ErrorInteraction
+  toobarFocusedRef: React.RefObject<boolean>
```

Comment:
Typo? toolbarFocusedRef?
There are 9 in this PR.

Reply:
nice catch. thanks!